client/state/rewind/test/selectors.js

```diff
@@ -107,3 +108,37 @@ describe( 'getBackupRetentionDays()', () => {
 		}
 	);
 } );
+describe( 'getBackupRetentionUpdateRequestStatus()', () => {
+	it.each( [
```

Comment:
Same as above for `it.each()`. I believe in this case it doesn't make the tests more readable and would recommend just having separate tests that explain what each test is doing.